docker/rtc_service/Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM python:3.8.8-alpine3.12 AS builder
+
+RUN adduser --disabled-password wirepas
+
+RUN apk add --no-cache \
+		gcc \
+		bash \
+		build-base \
+		make \
+		musl-dev \
+		elogind-dev \
+		python3-dev \
+		py3-gobject3 \
+		cairo-dev \
+		gobject-introspection-dev
+
+RUN python3 -m pip install wheel setuptools pkgconfig
+
+USER wirepas
+WORKDIR /home/wirepas
+
+COPY --chown=wirepas ./python_transport /home/wirepas/python_transport
+WORKDIR /home/wirepas/python_transport
+
+RUN ./utils/generate_wheel.sh
+
+USER wirepas
+
```

Comment:
Add a note here that this is a hack as dbus client is part of python_transport, but later they should be splitted